go.mod

```diff
@@ -14,6 +15,7 @@ require (
 	github.com/go-kit/log v0.2.0
 	github.com/gopherjs/gopherjs v0.0.0-20190812055157-5d271430af9f // indirect
 	github.com/gorilla/mux v1.8.0
+	github.com/guseggert/go-ds-dynamodb v0.0.0-20220304162827-e7d473330642
```

Comment:
Should I move this over to the IPFS org?

Reply:
yep, moving it to ipfs sounds good. The alternative is ipfs-shipyard which is also a fine choice here.